concepts/Testing/Testing.md

```diff
@@ -97,6 +103,24 @@ describe('UsersController', function() {
 
 });
 ```
+## Running tests
+
+In order to run your test using mocha, you'll have to use `mocha` in the command line and then pass as arguments the test your want to run, be sure to call bootstrap.test.js before the rest of your test like this `mocha test/bootstrap.test.js test/unit/**/*.test.js`
```

Comment:
- "pass as arguments the test **your** want to run", your -> you
- "before the rest of your **test** like", test -> tests